src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashCause.java

```diff
@@ -118,6 +105,17 @@ public String getBuildStartCommentId() {
   }
 
   public Map<String, String> getEnvironmentVariables() {
```

Comment:
maybe leave a comment explaining why we don't store it in a field?

Reply:
Done.